github/repos_releases_test.go

```diff
@@ -689,8 +689,8 @@ func TestRepositoriesService_DeleteReleaseAsset(t *testing.T) {
 func TestRepositoriesService_UploadReleaseAsset(t *testing.T) {
 	t.Parallel()
 	var (
-		defaultUploadOptions     = &UploadOptions{Name: "n"}
-		defaultExpectedFormValue = values{"name": "n"}
+		defaultUploadOptions     = &UploadOptions{Name: "n.txt"}
+		defaultExpectedFormValue = values{"name": "n.txt"}
```

Comment:
I'm not sure if we need to modify the test for the `UploadReleaseAsset` function.

Reply:
I'm also a bit uncomfortable with touching the test of another function when not strictly necessary.